app/controllers/salesforce/persons_controller.rb

```diff
@@ -13,16 +15,16 @@ def create
 
   def create_contact
     begin
-      Salesforce::Contact.create!(topic.user)
+      Salesforce::Contact.create!(user)
       render json: success_json
     rescue => e
       render json: { errors: [e.message] }, status: 422
     end
   end
 
-  def find_topic
-    params.require(:topic_id)
-    topic_id = params[:topic_id]
-    @topic = Topic.find_by(id: topic_id)
+  def find_user
+    params.require(:user_id)
+    user_id = params[:user_id]
+    @user = User.find_by(id: user_id)
```

Comment:
`User.find_by` may return `nil` if the params is invalid. We usually `raise Discourse::InvalidParameters.new(:user_id)` in cases like this.

Reply:
It's done :+1: